services/basic_types.proto

```diff
@@ -1745,3 +1750,32 @@ message PendingAirdropValue {
      */
     uint64 amount = 1;
 }
+
+/**
+ * Token claim airdrop<br/>
+ * Complete one or more pending transfers on behalf of the
+ * recipient(s) for an airdrop.
+ *
+ * The sender MUST have sufficient balance to fulfill the airdrop at the
+ * time of claim. If the sender does not have sufficient balance, the
+ * claim SHALL fail.<br/>
+ * Each pending airdrop successfully claimed SHALL be removed from state and
+ * SHALL NOT be available to claim again.<br/>
+ * Each claim SHALL be represented in the transaction body and
+ * SHALL NOT be restated in the record file.<br/>
+ * All claims MUST succeed for this transaction to succeed.
+ *
+ * ### Record Stream Effects
+ * The completed transfers SHALL be present in the transfer list.
+ */
+message TokenClaimAirdropTransactionBody {
+    /**
+     * A list of one or more pending airdrop identifiers.
+     * <p>
+     * This transaction MUST be signed by the account identified by
+     * the `receiver_id` for each entry in this list.<br/>
+     * This list MUST contain between 1 and 10 entries, inclusive.<br/>
+     * This list MUST NOT have any duplicate entries.
+     */
+    repeated PendingAirdropId pending_airdrops = 1;
+}
```

Comment:
The general pattern for Services is to add a new proto file for each transaction (e.g. `token_create`, `token_reject`, `token_update`, etc...).
This should _probably_ be in a file `token_claim_airdrop.proto`.

Reply:
Fixed 👍